scripts/workflows/e2e-matrix.py

```diff
@@ -15,8 +15,22 @@ def test_scripts(prefix):
 
 matrix = {
     "test": test,
-    "backend": ["replica"],
-    "os": ["macos-12", "ubuntu-20.04"]
+    "backend": ["pocketic", "replica"],
+    "os": ["macos-12", "ubuntu-20.04"],
+    "exclude": [
+        {
+            "backend": "pocketic",
+            "test": "dfx/bitcoin"
+        },
+        {
+            "backend": "pocketic",
+            "test": "dfx/canister_http"
+        },
+        {
+            "backend": "pocketic",
+            "test": "dfx/canister_logs"
+        }
```

Comment:
Note for a follow up PR: this should be reenabled once PocketIC is taken from the latest master commit (enabling canister logging).